### PR TITLE
Create Templates

### PR DESCRIPTION
**Pull request by erunks**
## What
Created some Issue templates and a Pull Request template

## Link to Issue
N/A

## Acceptance
| Issue template | Issue created | 
| --- | --- |
| image | https://github.com/erunks/speedrun.website/issues/1 |
| image | https://github.com/erunks/speedrun.website/issues/2 |
| image | https://github.com/erunks/speedrun.website/issues/3 |

### CODE REVIEW

**Comment by maxcaplan** [comment]
Everything looks good to me. However, I have rarely worked with templates...

**Comment by erunks** [comment]
> Everything looks good to me. However, I have rarely worked with templates...

First time trying the Issue Form templating for me, but they have an okay [documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema) of it, since it's in beta

**Review by zdylag — APPROVED** [review]
Thanks for testing it! In the future screenshots may also help things like this so we can see what it looks like to create.

**Review by maxcaplan — APPROVED** [review]
I'm going to approve this as I don't think we'll need anything else. Good work!

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    id: what-happened
+    attributes:
+      label: What happened?
+      placeholder: Tell us what you saw
+    validations:
+      required: true
+  - type: textarea
+    id: what-was-expected
+    attributes:
+      label: What did you expect to happened?
```
Comment (typwo) [inline .github/ISSUE_TEMPLATE/bug_report.yml:28]: Correct "happened" to "happen"

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Feature Request
+description: Suggest an idea to add support for to this project
```
Comment (typwo) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: Omit "to" or "for"

**Comment by typwo** [comment]
These look great. Thanks for putting them together.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (erunks) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Nice catch!

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (erunks) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: Nice catch, again!

**Pull request merged**